data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
             updatedBy: "Alexander", updatedAt: "2020-02-03",
         },
     },
+    {
+        meta: {
+            name: "12/06/2020 Olympia",
+            link: "https://www.startnext.com/12062020",
+            keywords: ["petition", "networking", "event", "grassroots"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "One-time event where up to 90,000+ people come together to rent the Olympia stadium Berlin to sign various petitions."
+            + " In Germany, petitions must be discussed by parliament if they get more than 50,000 signatures. In addition there will be talks and presentations on currently pressing social and environmental problems.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "German Zero",
+            link: "https://germanzero.de/",
+            keywords: ["policy", "legislation"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "GermanZero is creating a climate action plan to meet Germany's 1.5C commitment that will be turned into proposed legislation before 2022. A nation-wide campaign will then try to get that legislation passed in parliament.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Hamburg muss handeln",
+            link: "http://hamburg-muss-handeln.de/",
+            keywords: ["private-sector", "legislation", "local", "grassroots"],
+            label: "hamburg-muss-handeln",
+        },
+        description: {
+            content: "A coalition of 300+ companies and organizations from Hamburg to express support for the demands of Fridays for Future. Also provides advice for upcoming state elections.",
```

Comment:
This initiative is not a coalition of 300+ organizations. These just signed the open letter. 
+ 'advice for a vote for climate'

Reply:
Yes. I changed it to `An open letter addresses at local governments to embrace Fridays for Future's demands. 300+ companies and organizations from Hamburg signed it. Hamburg muss handeln also provides climate-focused advice for upcoming state elections.`.
Changes are reflected online, not in this PR as we discontinued this repository.

Reply:
ok great.